docs/specs/iota_did_method_spec.md

```diff
@@ -0,0 +1,324 @@
+
+## Introduction
+
+### The IOTA Tangle
+
+This specification defines a method of implementing DID on top of the IOTA Tangle, which is a Distributed Ledger Technology (DLT) using a Tangle data structure. In contrast to a Blockchain, the Tangle does not store messages in blocks and chain them together, but rather creates a data structure where a message references between one and eight previous messages, creating a parallel structure. 
+
+For this method, the most important features of IOTA are: 
+
+* The lack of fees, requiring no cryptocurrency tokens to be owned in order to submit a message to the DLT.
+* The DLT has a public and permissionless network which runs the IOTA cryptocurrency.
+* Pure data messages are possible to be stored immutably. 
+* Few nodes store the entire Tangle, requiring additional logic to prove the immutability of data. 
+
+## DID Method Name
+
+The namestring to identify this DID method is: `iota`.
+
+A DID that uses this method MUST begin with the following prefix: `did:iota`. Following the generic DID specification, this string MUST be completely in lowercase.
+
+## DID Format
+
+The DIDs that follow this method have the following format:
+```
+iota-did = "did:iota:" iota-specific-idstring
+iota-specific-idstring = [ iota-network ":" ] [ network-shard ":" ] iota-tag
+iota-network = char{,6}
+network-shard = char{,6}
+iota-tag = base-char{44}
+char = 0-9 a-z
+base-char = 1-9 A-H J-N P-Z a-k m-z
+```
+
+### IOTA-Network
+
+The iota-network is an identifer of the network where the DID is stored. This network must be an IOTA Tangle, but can either be a public or private network, permissionless or permissioned.
+
+The following values are reserved and cannot reference other networks:
+1. `main` references the main network which refers to the Tangle known to host the IOTA cryptocurrency
+2. `dev` references the test network known as "devnet" or "testnet" maintained by the IOTA Foundation.
+
+Note that when no IOTA network is specified, it is assumed that the DID is located on the `main` network. This means that the following DIDs will resolve to the same DID Document:
+```
+did:iota:main:H3C2AVvLMv6gmMNam3uVAjZpfkcJCwDwnZn6z3wXmqPV
+did:iota:H3C2AVvLMv6gmMNam3uVAjZpfkcJCwDwnZn6z3wXmqPV
+```
+
+### Network-Shard
+
+In a future update to the network, IOTA will likely have shards containing subsets of the Tangle. While this is currently not the case, the space in the DID is reserved and implementations should be able to handle this extra field. The value can currently be ignored. 
```

Comment:
Should this section also mention that if a shard is specified then the network cannot be elided? e.g. `did:iota:main:shard`

Reply:
Good point, will add.